src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_container_rm_scenarios.py

```diff
@@ -0,0 +1,66 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.profiles import ResourceType
+from azure.cli.testsdk import ScenarioTest, api_version_constraint, ResourceGroupPreparer, StorageAccountPreparer
+from azure_devtools.scenario_tests import AllowLargeResponse
+
+
+@api_version_constraint(ResourceType.MGMT_STORAGE, min_api='2019-06-01')
+class StorageContainerRmScenarios(ScenarioTest):
+    @AllowLargeResponse()
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer()
+    def test_storage_container_using_rm_main_scenario(self):
+        # 1. Test create command.
+
+        # Create container with specified public access and metadata.
+        cname = self.create_random_name('container', 24)
+        self.kwargs.update({
+            'cname': cname,
+        })
+        result = self.cmd('storage container-rm create --account-name {sa} -g {rg} '
```

Comment:
also need to test fail on exist

Reply:
A test after this one is designed to test fail on exist